wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/ConversationMessageCell.swift

```diff
@@ -159,6 +173,10 @@ protocol ConversationMessageCellDescription: AnyObject {
 
 extension ConversationMessageCellDescription {
 
+    var supportsActions: Bool {
```

Comment:
question: I thought you removed `supportsActions`, do we still need it here?

Reply:
I didn't remove it, but since it the result was mostly `false` I created a default implementation returning `false`.